lib/susedistribution.pm

```diff
@@ -198,6 +198,7 @@ sub become_root {
     my ($self) = @_;
 
     $self->script_sudo('bash', 0);
+    wait_still_screen;
```

Comment:
wow, I misjudged how many wait_serials we need *after* the failure :)

But https://openqa.suse.de/tests/569123#step/smt/11 has your root catched, so longer wait_serial should do